.github/workflows/integration.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           done
           docker logs elastic
           echo "========================== loki ========================="
-          while ! curl -sX GET "http://127.0.0.1:3100/loki/api/v1/ready"
+          while ! curl -sX GET "http://127.0.0.1:3100/ready" | grep -q "^ready$"
```

Comment:
Did this commit slip in by mistake? I don't object to it, it just seems out of place.